temoa/model_checking/element_checker.py

```
@@ -33,102 +8,151 @@ class to hold members of "validation sets" used by loader to validate elements a
+    This will NOT match:
+      - ('a', 4) (non-exception part (4,) is not in the valid set)
+      - ('cat', 3) (non-exception part (3,) is not in the valid set)
+    """
+
+    # Stored for reference; these are examples of common automatic approvals.
+    REGION_REGEXES: list[str] = [
         r'\+',  # any grouping with a plus sign
         r'^global\Z',  # the exact word 'global' with no leader/trailer
     ]
 
     def __init__(
```

Ruff (0.14.1)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)